spyder/plugins/ipythonconsole/widgets/help.py

```diff
@@ -43,9 +43,10 @@ def get_documentation(self, content):
         text = data.get('text/plain', '')
         signature = self.get_signature(content).split('(')[-1]
         if text:
-            documentation = text.split(signature)
-            if len(documentation) > 1:
-                return documentation[-1]
+            if signature:
+                documentation = text.split(signature)
+                if len(documentation) > 1:
+                    return documentation[-1]
```

Comment:
I think this would avoid showing documentation for objects without signature, right? I mean, if there's no signature we should still show the documentation (if `text` has some content).